zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/test/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProviderConfigTest.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.zookeeper.metrics.prometheus;
+
+import io.prometheus.client.CollectorRegistry;
+import org.apache.zookeeper.metrics.MetricsProviderLifeCycleException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Properties;
+
+/**
+ * @author hezhangjian
+ */
+public class PrometheusMetricsProviderConfigTest {
+
```

Comment:
this unit test is great. we also need to write some cases for the valid address and port?

Reply:
I think the PrometheusMetricsProviderTest is already a valid test case.

Reply:
Added a Valid host and port test. The abality test should be tested in PrometheusMetricsProviderTest.

I am not sure if test a specify port is properly, so i use port 0 for test